crates/optimism/cli/src/commands/import_receipts.rs

```diff
@@ -270,7 +270,6 @@ mod test {
     /// No receipts for genesis block
     const EMPTY_RECEIPTS_GENESIS_BLOCK: &[u8] = &hex!("c0");
 
-    #[ignore]
```

Comment:
:tada:

Reply:
wasn't supposed to be removed! good that you brought my attention to this